agent/consul/server.go

```diff
@@ -468,8 +472,17 @@ func NewServer(config *Config, flat Deps, externalGRPCServer *grpc.Server) (*Ser
 
 	// TODO(NET-1380, NET-1381): thread this into the net/rpc and gRPC interceptors.
 	s.incomingRPCLimiter = rpcRate.NewHandler(rpcRate.HandlerConfig{
-		// TODO(NET-1379): pass in _real_ configuration.
-		GlobalMode: rpcRate.ModePermissive,
+		// TODO(NET-1379):
+		// - should disabled mode not register this handler?  Or does the handler have logic for disabled?
+		GlobalMode: rpcRate.Mode(config.RequestLimitsMode),
+		GlobalReadConfig: multilimiter.LimiterConfig{
+			Rate:  config.RequestLimitsReadRate,
```

Comment:
Handler is always configured.  Here we detect the user did not set the mode and we pass rate.inf.